src/demo/rewireEcommerceApiService.js

```diff
@@ -1,21 +1,40 @@
 import EcommerceApiService from '../../src/data/services/EcommerceApiService';
 
-import { codes, coupons } from './data';
+import { codes, codesCsv, coupons } from './data';
 
-const firstCouponHasError = coupons.results[0].has_error;
+const firstCouponHasError = coupons[0].has_error;
 
 const findCouponIndexById = couponId =>
-  coupons.results.findIndex(coupon => coupon.id === couponId);
+  coupons.findIndex(coupon => coupon.id === couponId);
 
 const rewire = () => {
-  EcommerceApiService.fetchCouponOrders = () => Promise.resolve({
-    data: coupons,
-  });
+  const fetchData = (options = { page: 1 }, couponData) => {
```

Comment:
Can we move the `fetchData` function outside of the `rewire` function? We should try to keep the `rewire` function to contain only the actual rewirings of the service functions, and not any helper functions.